src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPlugin.kt

```diff
@@ -26,7 +28,20 @@ open class MavenPublishPlugin : Plugin<Project> {
 
     val pom = MavenPublishPom.fromProject(p)
     p.setCoordinates(pom)
-    p.configureTargets(extension)
+    p.checkProperties()
+    p.configureArchivesTasks()
+
+    p.publishing.repositories.maven { repo ->
+      repo.name = "mavenCentral"
+      repo.setUrl("https://oss.sonatype.org/service/local/staging/deploy/maven2/")
+      repo.credentials(PasswordCredentials::class.java)
+
+      p.afterEvaluate {
+        if (it.version.toString().endsWith("SNAPSHOT")) {
+          repo.setUrl("https://oss.sonatype.org/content/repositories/snapshots/")
+        }
+      }
+    }
```

Comment:
This will move again with the new API. Before the repo was just called maven but I think it's more clear if Gradle's tasks are called `publish...ToMavenCentralRepository` and not `publish...ToMavenRepository`